bundles/org.openhab.binding.modbus.stiebeleltron/src/main/resources/ESH-INF/i18n/modbus.stiebeleltron_xx_XX.properties

```diff
@@ -0,0 +1,17 @@
+# FIXME: please substitute the xx_XX with a proper locale, ie. de_DE
```

Comment:
Did this file find its way into your binding by intention?

Reply:
File removed

Reply:
Again with the unchecked exception. The user could configure an unsupported unit.

Reply:
I guess this comment goes to StiebelEltronHandler (and not to the .properties file)